ClangSharp.PInvokeGenerator/Cursors/Exprs/BinaryOperator.cs

```diff
@@ -59,118 +59,23 @@ protected override CXChildVisitResult VisitChildren(CXCursor childHandle, CXCurs
         {
             ValidateVisit(ref handle);
 
-            Expr expr;
-
-            switch (childHandle.Kind)
+            if (childHandle.IsExpression)
```

Comment:
The logic is much more simplified now, and supports more. We know that a `BinaryOperator` will contain two children and they will both be `expressions`. The type of those expressions could be just about anything, so we really just want to check if it is one rather than check each type here.